### ISSUE TITLE
New CPU utilization issue

### ISSUE THREAD

**Issue by j-mikhail** [Quicksaver/Tab-Groups#480]
Hi Luís!

I'm having a new issue with Tab Groups. Running v2.1 on Firefox 49.0.2. I noticed that when I first launched Firefox and it restored my tabs and tab groups, despite only having one active tab (and the rest still unloaded), Firefox was already taking up 30% of my CPU:

![cpu](https://cloud.githubusercontent.com/assets/3420913/20196668/b96d78f8-a769-11e6-81a7-16bc4f83b8ed.png)

So I did some poking around, and according the the Firefox performance manager, Tab Groups is the culprit:

![performance](https://cloud.githubusercontent.com/assets/3420913/20196681/ceca4d16-a769-11e6-951a-a634be5aea41.png)

I am not sure what's causing this. Is there additional data I can provide? Thanks!

**Comment by Quicksaver** [comment]
First to clarify, does the high CPU happen immediately when starting Firefox, or only after you click the groups toolbar button or access the manage groups view for the first time?

Does this happen every time you start Firefox? If it does, does it happen if you disable all your other add-ons except for Tab Groups?

**Comment by j-mikhail** [comment]
Yes, immediately, before I even interact with a single tab.

It's begun happening since the update to 49.0.2, and it does happen every time. I have not tried disabling my other add-ons since the Performance Manager said they were performing well, but I'll give it a try and let you know.

**Comment by Quicksaver** [comment]
It's not about them performing well or not, but about there possibly being some kind of incompatibility with them that's causing Tab Groups to hog CPU. ;) Please do give it a try and let me know.

**Comment by j-mikhail** [comment]
OK, so I disabled all of my add-ons, and started reactivating them one-by-one, restarting Firefox each time. Having done so, I have now successfully reactivated them all, and the utilization issue is no longer occurring. So I'm extra confused about why it was occurring in the first place. 😕 However, I'm next going to start reactivating some tabs to see if one of them may have been the catalyst somehow...

**Comment by Quicksaver** [comment]
This is why this bug is so weird... I have had reports about it before, and I have seen it myself once (on Linux only) but by the time I tried to do some debugging it went away. It's hard to get to the root of the problem.

Let's try a different strategy if this starts happening again. Don't disable your add-ons, unless you really need to use your browser at the time of course ;), try disabling them one at a time instead to see if you can at least pinpoint which one makes it run correctly again.

**Comment by j-mikhail** [comment]
Will do. I'll post back when I have more data!
